app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -347,15 +356,24 @@ def parse_persistent_volume(persistent_volume)
       new_result = parse_base_item(persistent_volume)
       new_result.merge!(parse_volume_source(persistent_volume.spec))
       new_result.merge!(
-        :type           => 'PersistentVolume',
-        :parent_type    => 'ManageIQ::Providers::ContainerManager',
-        :capacity       => parse_resource_list(persistent_volume.spec.capacity.to_h),
-        :access_modes   => persistent_volume.spec.accessModes.join(','),
-        :reclaim_policy => persistent_volume.spec.persistentVolumeReclaimPolicy,
-        :status_phase   => persistent_volume.status.phase,
-        :status_message => persistent_volume.status.message,
-        :status_reason  => persistent_volume.status.reason
+        :type                    => 'PersistentVolume',
+        :parent_type             => 'ManageIQ::Providers::ContainerManager',
+        :capacity                => parse_resource_list(persistent_volume.spec.capacity.to_h),
+        :access_modes            => persistent_volume.spec.accessModes.join(','),
```

Comment:
@simon3z ```desired_access_modes``` and ```actual_access_modes``` are indeed parsed and saved here as arrays. On the UI side ```join``` has to be used just to convert the array into string before displaying it (can use ```to_s``` instead). Please note that the line above was not added in this PR and has to be changed as part of the refactor of ```PersistentVolume``` as @alongoldboim said (this PR is only adding ```PersistentVolumeClaim```).